cookbooks/aws-parallelcluster-config/templates/default/init/fetch_and_run.erb

```diff
@@ -15,6 +15,7 @@ function error_exit () {
   script=`basename $0`
   echo "parallelcluster: ${script} - $1"
   logger -t parallelcluster "${script} - $1"
+  echo "${script} - $1." "Note: If --rollback-on-failure was set to false, check /var/log/cfn-init.log and /var/log/cloud-init-output.log for more details." > /var/log/parallelcluster/headnode_bootstrap_error_msg
```

Comment:
nit: can we remove `fetch_and_run -`? Users doesn't know what is fetch_and_run, it is a function name in our code and I think adding it doesn't help with better understanding the problem if we have it in the cloudformation. And can we also remove "Note:" 
```
"$1. If --rollback-on-failure was set to false, check /var/log/cfn-init.log and /var/log/cloud-init-output.log for more details."
```